Author: Steffen Moeller <moeller@debian.org>
Description: Adjusting paths.
Disclaimer: This is mostly dead code in upstream.
--- a/stripchart/stripchart.cnf
+++ b/stripchart/stripchart.cnf
@@ -8,7 +8,7 @@
 # Directory where gnuplot is
 # $gnuplot = "/usr/local/gnuplot-3.7";
 #$gnuplot = "/disks/asimov/a/users/hiramc/local/src/gnuplot-3.7";
-$gnuplot = "/usr/local/bin"
+$gnuplot = "/usr/bin"
  
 # Temporary files
 $suffix = rand(10000);
@@ -42,7 +42,7 @@
 $tzdiff = timegm($sec,$min,$hour,$mday,$mon,$year) - timelocal($sec,$min,$hour,$mday,$mon,$year);
 
 # Where is the stripchart executable located?
-$stripchartexe = "./stripchart"
+$stripchartexe = "/usr/lib/cgi-bin/stripchart"
  
 # What is the default number of stripcharts?
 $defaultnumcharts = 1;
@@ -52,11 +52,11 @@
 
 # Where is the list of datafiles for stripchart.cgi?
 # Note: there is a sample copy in the samples directory: 
-$datafilelist = "datafiles";
+$datafilelist = "/usr/share/stripchart/datafiles";
  
-# Where is the list of user-definied cgi queries?
+# Where is the list of user-defined cgi queries?
 # Note: this file gets created by the cgi - must put it somewhere that the cgi user can write to 
-$queryfilelist = "querylist";
+$queryfilelist = "/tmp/querylist";
 
 # What time is it right now?
 $rightnow = time;
--- a/stripchart/stripchart
+++ b/stripchart/stripchart
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 
 # The contents of this file are subject to the Mozilla Public License
 # Version 1.0 (the "License"); you may not use this file except in
@@ -47,7 +47,7 @@
 ################
 
 # Where is the .cnf file?
-$cnfpath = "./stripchart.cnf";
+$cnfpath = "/etc/stripchart.cnf";
 
 # Read it in:
 open (CNFFILE,$cnfpath) or die "cannot open configuration file: $cnfpath\nmake sure this variable is set properly";
--- a/stripchart/stripchart.cgi
+++ b/stripchart/stripchart.cgi
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#!/usr/bin/perl
 
 # The contents of this file are subject to the Mozilla Public License
 # Version 1.0 (the "License"); you may not use this file except in
@@ -44,7 +44,7 @@
 ################
  
 # Where is the .cnf file? 
-$cnfpath = "./stripchart.cnf";
+$cnfpath = "/etc/stripchart.cnf";
 
 # Read it in: 
 open (CNFFILE,$cnfpath) or die "cannot open configuration file: $cnfpath\nmake sure this variable is set properly"; 
